apps/election-manager/src/components/ExportElectionBallotPackageModalButton.tsx

```diff
@@ -34,10 +34,14 @@ const USBImage = styled.img`
 `;
 
 export function ExportElectionBallotPackageModalButton(): JSX.Element {
-  const { electionDefinition, usbDriveStatus, usbDriveEject } = useContext(
-    AppContext
-  );
+  const {
+    electionDefinition,
+    usbDriveStatus,
+    usbDriveEject,
+    currentUserSession,
+  } = useContext(AppContext);
   assert(electionDefinition);
+  assert(currentUserSession); // TODO(auth) should this make sure we have an admin
```

Comment:
Why not `assert(currentUserSession?.type === 'admin')`? What needs to happen before we can do that? This comment isn't really clear about that.

Reply:
I'm a little confused where we are currently at with some of the auth stuff, and I think we are planning to add another role (clerk), so I just thought it was better to make a searchable TODO for these to re-evaluate when we have a clear auth structure in the apps rather then trying to figure it all out now.